lib/keystone.lua

```diff
@@ -109,26 +108,25 @@ function Client:_updateToken(callback)
     end
 
     local function handleTokenResponse(res)
-      local data = ''
+      local chunks = {}
       res:on('data', function(chunk)
-        data = data .. chunk
+        table.insert(chunks, chunk)
```

Comment:
Hi Tomaz... It's faster to append to a table. String are immutable in lua so the concatenation creates a copy then appends. It's an optimization and cleanup I have been making in various places.